formats/host.ps1xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/PowerShell/PowerShell/master/src/Schemas/Format.xsd">
+<Configuration>
     <ViewDefinitions>
         <View>
-            <Name>XoPowershell.Host</Name>
+            <Name>XoHost</Name>
```

Comment:
Please make sure the type names are consistent across all PSObject types. Also, why the change?